dask/array/tests/test_fft.py

```diff
@@ -195,3 +195,9 @@ def test_wrap_fftns(modname, funcname, dtype):
         wfunc(darr2c, (darr2c.shape[0] - 1, darr2c.shape[1] - 1), (0, 1)),
         func(nparrc, (nparrc.shape[0] - 1, nparrc.shape[1] - 1), (0, 1))
     )
+
+
+@pytest.mark.parametrize("n", list(range(1, 10)))
+@pytest.mark.parametrize("d", [1.0, 0.5, 2 * np.pi])
```

Comment:
- Do we really need to check this many `n`?
- Slight preference to move these loops inside the test function. Up to you though.